[docs] highlight importance of context="module" (#2131)
sveltejs/kit: documentation/docs/03-loading.md

	}
</script>
```

> > Note the `<script context="module">` — this is necessary because `load` runs before the component is rendered. Your "regular" code should go into a second `<script>` tag.  

Reviewer: ```suggestion
> Note the `<script context="module">` — this is necessary because `load` runs before the component is rendered. Code that is per-component instance should go into a second `<script>` tag.  
```
